bash/terraform/terraform_run.functions.md

```diff
@@ -239,10 +246,10 @@ changes, and also to ensure that the git audit info is accurate.
     terraform_preinit() {
         export TF_VAR_host_size="humungous"
         export TF_VAR_secret="$(get_secret)"
-    
+
         required_vars "TF_VAR_host_size TF_VAR_secret" || return 1
     }
-    
+
```

Comment:
I think this whitespace is needed to auto-generate this part of the docs correctly